src/hotspot/cpu/aarch64/templateTable_aarch64.cpp

```diff
@@ -3778,7 +3778,7 @@ void TemplateTable::instanceof() {
   __ push(atos); // save receiver for result, and for GC
   call_VM(r0, CAST_FROM_FN_PTR(address, InterpreterRuntime::quicken_io_cc));
   // vm_result_2 has metadata result
-  __ get_vm_result_2(r0, rthread);
+  __ get_vm_result_metadata(r0, rthread);
```

Comment:
Stale comment just above this line. Grep around for `vm_result_2` to look for other cases?